tests/foreman/cli/test_host.py

```diff
@@ -1014,12 +1015,20 @@ def test_positive_create_using_libvirt_without_mac(self):
 
         :CaseImportance: Critical
         """
+        new_loc = make_location()
         compute_resource = entities.LibvirtComputeResource(
             url='qemu+ssh://root@{0}/system'.format(
                 settings.compute_resources.libvirt_hostname
-            )
+            ),
+            organization=[self.new_org['id']],
+            location=[new_loc['id']]
         ).create()
-        host = entities.Host()
+        entities.Organization(id=self.new_org['id']).read()
```

Comment:
@jhutar why we need to read org and loc here?

Reply:
+1 - this seems to be not necessary

Reply:
At some point I thought this is important for org to see CR, but tested now and it works without it. Removed. Thanks!